app/src/utils/MadLibs.ts

```diff
@@ -29,6 +29,18 @@ function getMadLibPhraseText(madLib: MadLib): string {
   return madLibText;
 }
 
+const VARIABLES = {
```

Comment:
I think we could more strongly type this Record<string, VariableId>

Reply:
Good call!